src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -204,6 +204,10 @@ public sealed partial class ComAliasNameAttribute : System.Attribute
         public ComAliasNameAttribute(string alias) { }
         public string Value { get { throw null; } }
     }
+    public static class CollectionsMarshal
```

Comment:
nit: add `partial`

We should also try to auto-generate this if possible.

@safern might be able to help provide guidance here on how for local build of coreclr (outside of https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/updating-ref-source.md)